packages/bpk-component-graphic-promo/src/custom-breakpoints.module.scss

```diff
@@ -0,0 +1,74 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// FIXME: Replace the custom breakpoints with on es from Backpack when ready. (ST-4102)
+
+@function pxtorem($px) {
+  @return ($px / 16) * 1rem;
+}
+
+$breakpoint-landscape-desktop-min: pxtorem(801);
+$breakpoint-landscape-tablet-max: pxtorem(800);
+$breakpoint-landscape-tablet-min: pxtorem(701);
+$breakpoint-portrait-large-max: pxtorem(700);
+$breakpoint-portrait-large-min: pxtorem(513);
+$breakpoint-portrait-medium-max: pxtorem(512);
+$breakpoint-portrait-medium-min: pxtorem(425);
+$breakpoint-portrait-small-max: pxtorem(424);
```

Comment:
Are we able to just define the rem value instead of doing a conversion?

Might also be worth seeing if we can make this up based on existing spacing values?

As looking at this it just looks like a magic number and might cause confusion to future contributors or maintainers.

If you need the maths to do px to rem conversion its pixel value / 16 = rem value